docs/Releasing-the-Distribution.md

```diff
@@ -1,83 +1,7 @@
-- [OpenSearch Release Process](#opensearch-release-process)
```

Comment:
Can we still keep these heading mappings without removing them? So that if we have pull back from `docs/` folder we still have the headings in markdown format.

Reply:
It is taking up a lot of space (almost 1.5 page) Also if we need to revert back, just reverting this PR should restore it. The wiki auto formats the table of contents 
![image](https://github.com/opensearch-project/opensearch-build/assets/61760125/86190eff-c99a-4e55-a92b-bfd183baf007)